### DISCUSSION TITLE
remind user to upgrade packages

### DISCUSSION THREAD

**Discussion by EricWVGG** [gatsbyjs/gatsby#34415] [Ideas / Feature Requests]
I think it'd be really neat if `gatsby develop` would remember the last time a user ran package updates (I use `yarn upgrade-interactive --latest`) and periodically reminded the user, "30 days since you last updated Gatsby" or whatever.

**Comment by AnilSeervi** [comment]
Change the `develop` command in package.json to be
```shell
yarn upgrade-interactive --latest && gatsby develop
```

this way it'll check everytime you run `yarn develop`

**Comment by EricWVGG** [comment]
Well that sounds just awful, but upon further reflection this idea is outside the demesne of Gatsby, so I'll just retract this.
